tests/e2e/stepDefinitions/registerContext.js

```diff
@@ -13,34 +17,35 @@ Given("user has navigated to the Setup page", async function () {
 });
 
 When(
-  "user registers with username {string} ,password {string} and confirm password {string}",
+  "user sign up with username {string}, password {string} and confirm password {string}",
   async function (username, password, cPassword) {
     await registerPage.submitRegisterData(username, password, cPassword);
   }
 );
 
 Then("user should see message {string}", async function (message) {
-  let a = await registerPage.getErrorMessage(message);
+  let actualErrorMessage = await registerPage.getErrorMessage(message);
+  let expectedErrorMessage = await message.split(",");
   assert.deepEqual(
-    a[0],
-    a[1],
-    `Expected to get error but got diffrent errors: ${a[0]}`
+    actualErrorMessage,
+    expectedErrorMessage,
+    `Expected to get ${expectedErrorMessage} but got diffrent errors: ${actualErrorMessage}`
   );
 });
 
 Then("user should redirect to sign in page", async function () {
   assert.equal(
     page.url(),
-    await registerPage.loginURL,
+    await loginPage.loginPageUrl,
     `Expected user to redirect to login page but redirected to page with url ${page.url()}`
   );
 });
 
 Then("user should get success message {string}", async function (succesMsg) {
-  let errors = await registerPage.getSuccessMsg();
+  let errors = await registerPage.getSuccessMsg(succesMsg);
```

Comment:
Better using meaningful variable names.
```suggestion
  const actualMessage = await registerPage.getSuccessMsg(succesMsg);
```